wallet/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN apt-get update && apt-get install -y wget
 
 # Install EOSIO
 RUN wget -O eosio.deb $EOSIO_PACKAGE_URL
-RUN apt-get install -y /eosio.deb
+RUN apt-get install -y /eosio.deb --no-install-recommends && apt-get clean \
```

Comment:
@JustinCast the make run it's still working after add this change? could you remove your local container and run `make run` from the scratch? just to confirm that the wallet it's still running

Reply:
Yep, it's still running